core/lib/presentation/views/html_viewer/html_content_viewer_on_web_widget.dart

```diff
@@ -104,8 +104,6 @@ class _HtmlContentViewerOnWebState extends State<HtmlContentViewerOnWeb> {
            } else if (textContent && isValidUrl(textContent)) {
               window.parent.postMessage(JSON.stringify({"view": "$createdViewId", "type": "toDart: OpenLink", "url": "" + textContent}), "*");
               e.preventDefault();
-           } else {
-              e.preventDefault();
```

Comment:
Could you please explain more about this change?

Reply:
When you click tag <a/> of HTML, function onClickListener cant handle tag <a/>, this function will handle child of tag <a/>. So code will run to line 107, but it this line you set `e.preventDefault()`. It reason no action will cotinue when click.

Reply:
@ManhNTX

I've fetched your code up and running and some emails can't open in a new tab. Please try to double check with emails with website content more closely. See video below for details.

https://user-images.githubusercontent.com/80730648/177456238-daadeefb-85f8-4e6d-82fb-be1c7c0b7181.mov

Reply:
This error appears on `https://jmap.linagora.com` but on `https://gateway.upn.integration-open-paas.org` not.